docs/user-guide/ze-usage.md

```diff
@@ -6,23 +6,24 @@ Review this section to familiarize yourself with Zowe Explorer and make the best
 
 ### Operating systems
 
-- Windows 10+
+
 - MacOS 10.15 (Catalina), 11 (Big Sur), 12 (Monterey)
 - Unix-like:
    - [CentOS](https://www.centos.org/) 8+
    - [RHEL](https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux) 8+
    - [Ubuntu](https://ubuntu.com/) 20.04+
+- Windows 10+
 
-### Integrated development environments:
+### Integrated development environments: 
 
-- [Theia](https://theia-ide.org/) 18+
-- [Red Hat CodeReady Workspaces](https://www.redhat.com/en/technologies/jboss-middleware/codeready-workspaces)
 - [Eclipse Che](https://www.eclipse.org/che/)
+- [Red Hat CodeReady Workspaces](https://www.redhat.com/en/technologies/jboss-middleware/codeready-workspaces)
+- [Theia](https://theia-ide.org/) 18+
```

Comment:
Theia has not reached version 18 yet, I think this should say `1.18+` to match the ZE readme: https://github.com/zowe/vscode-extension-for-zowe/blob/cea37e20cf41dc4073c63f7c0bb652a019472bfe/packages/zowe-explorer/README.md?plain=1#L31

Reply:
Should we also add a note similar to the one we recently added to the ZE readme to recommend community releases?
> Zowe Explorer is compatible only with Theia 1.18.0 or higher. We recommend using a [Theia community release](https://theia-ide.org/releases/) as Zowe Explorer could experience possible unexpected behaviors with the latest Theia releases.

Reply:
Updated the file to reflect both comments -- thank you!